src/dotnet-interactive-vscode-insiders/package.json

```
@@ -206,6 +206,38 @@
         "command": "dotnet-interactive.acquire",
         "title": "Install .NET Interactive"
       },
+      {
+        "command": "dotnet-interactive.createNewInteractive",
+        "title": "Polyglot Notebook: Create Interactive Window"
+      },
+      {
+        "command": "polyglot-notebook.openNotebook",
+        "title": "Polyglot Notebook: Open notebook"
+      },
+      {
+        "command": "polyglot-notebook.saveAsNotebook",
+        "title": "Polyglot Notebook: Save notebook as..."
+      },
+      {
+        "command": "polyglot-notebook.newNotebook",
+        "title": "Polyglot Notebook: Create new blank notebook"
+      },
+      {
+        "command": "polyglot-notebook.restartCurrentNotebookKernel",
+        "title": "Polyglot Notebook: Restart the current notebook's kernel"
+      },
+      {
+        "command": "polyglot-notebook.stopCurrentNotebookKernel",
+        "title": "Polyglot Notebook: Stop the current notebook's kernel"
+      },
+      {
+        "command": "polyglot-notebook.stopAllNotebookKernels",
+        "title": "Polyglot Notebook: Stop all notebook kernels"
+      },
+      {
+        "command": "polyglot-notebook.shareValueTo",
+        "title": "Share value to..."

```

Does this read more clearly?

```suggestion
        "title": "Share value with..."

```